package.json

```diff
@@ -112,20 +112,20 @@
     "express-logging": "1.1.1",
     "extract-zip": "2.0.1",
     "fastest-levenshtein": "1.0.16",
-    "fastify": "4.28.1",
+    "fastify": "4.29.0",
     "find-up": "7.0.0",
     "flush-write-stream": "2.0.0",
     "folder-walker": "3.2.0",
     "from2-array": "0.0.4",
     "fuzzy": "0.1.3",
-    "get-port": "5.1.1",
+    "get-port": "7.1.0",
     "gh-release-fetch": "4.0.3",
     "git-repo-info": "2.1.1",
     "gitconfiglocal": "2.1.0",
     "http-proxy": "1.18.1",
     "http-proxy-middleware": "2.0.7",
     "https-proxy-agent": "7.0.6",
-    "inquirer": "6.5.2",
+    "inquirer": "8.0.0",
```

Comment:
GitHub hid the comments: https://github.com/netlify/cli/pull/7019#discussion_r1941426419